rebar.config.erl

```diff
@@ -52,13 +52,17 @@ overrides() ->
     [ {add, [ {extra_src_dirs, [{"etc", [{recursive,true}]}]}
             , {erl_opts, [{compile_info, [{emqx_vsn, get_vsn()}]}]}
             ]}
-    , {add, snabbkaffe,
-       [{erl_opts, common_compile_opts()}]}
-    ] ++ community_plugin_overrides().
+    ] ++ snabbkaffe_overrides() ++ community_plugin_overrides().
 
 community_plugin_overrides() ->
     [{add, App, [ {erl_opts, [{i, "include"}]}]} || App <- relx_plugin_apps_extra()].
 
+%% Temporary workaround for a rebar3 erl_opts duplication
+%% bug. Ideally, we want to set this define globally
+snabbkaffe_overrides() ->
```

Comment:
Turns out the rebar3 bug that duplicates the defines runs pretty deep, and there were a few unsuccessful attempts to fix it in the upstream. Giving up on the general solution for now.